src/EditorFeatures/Core/Remote/SolutionChecksumUpdater.cs

```diff
@@ -141,11 +141,9 @@ private async Task SynchronizePrimaryWorkspaceAsync(CancellationToken cancellati
 
             using (Logger.LogBlock(FunctionId.SolutionChecksumUpdater_SynchronizePrimaryWorkspace, cancellationToken))
             {
-                var checksum = await solution.State.GetChecksumAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
pointless double computation.  TryInvokeAsync already has to do this to get the solution checksum passed in the PinnedSolutionInfo that is passes into the lambda.